lib/internal/console/constructor.js

```diff
@@ -381,9 +386,14 @@ const consoleMethods = {
 
 
   warn(...args) {
-    this[kWriteToConsole](kUseStderr, this[kFormatForStderr](args));
+    const color = typeof args === 'string' ? lazyUtilColors().yellow : null;
```

Comment:
```suggestion
    const color = ArrayPrototypeEvery(args, arg => typeof arg === 'string') ? lazyUtilColors().yellow : null;
```

Reply:
Good catch. Just realise we can handle `number` type as well.

Reply:
So, i think `typeof arg !== 'object' || typeof arg !== 'symbol'` would be great choice.

Reply:
I think we should stick to `typeof arg === 'string'` for now as it's the most likely to get consensus. We should also make sure that it's easy to decipher from e.g. `console.log(1);console.warn('1')` which one is a number and which one is a string.

Reply:
Any reason for not handling the `number` ?

Reply:
We would want to handle bigints the same way we do handle numbers. Currently, both are always shown in yellow; if we change that we should change in a way that's somehow consistent with `console.log` (maybe by changing `console.log` behavior, maybe numbers should be e.g. bold (and/or italic) instead of yellow), and I feel it should be a separate discussion.

Reply:
okay.